### DISCUSSION TITLE
How to add Python@3.11 to Path

### DISCUSSION THREAD

**Discussion by quackerex** [Homebrew/discussions#5154] [Everyday usage] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

Node@18 formula have this information about adding it to `PATH` so I can use the `node` command.
```
If you need to have node@18 first in your PATH, run:
  echo 'export PATH="/opt/homebrew/opt/node@18/bin:$PATH"' >> ~/.zshrc
```
Is there a similar way to add `python@3.11` path so `python3` links to `python@3.11`?

**Comment by ZhongRuoyu** [comment, marked as answer]
No, but you can do any of the below:

- Use `python3.11` (which should already be in `PATH`).
- Add `/opt/homebrew/opt/python@3.11/libexec/bin` (more generally, `"$(brew --prefix python@3.11)"/libexec/bin`) to `PATH`; symlinks like `python`, `python3`, `pip`, `pip3` can be found there.

The caveats in the output of `brew info python@3.11` should give you more info.
